co.uk.studiopond.spiderbro.metainfo.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>co.uk.studiopond.spiderbro</id>
+  
+  <name>Spiderbro</name>
+  <summary>An adventure puzzle game starring a purple spider!</summary>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0+</project_license>
```

Comment:
Where is the source code hosted at?

Reply:
and why not building from source?

Reply:
Hi. Source code can be found at https://github.com/pforpond/Spiderbro/blob/main/flatpak/vame%20(spiderbro%20flatpak).bas I'm not building from source as my code is written in basic and to compile it would require QB64. In my view it would make the install process unnecessarily complex for myself and unnecessarily long for the user. I can probably figure out how to do it but I'm not exactly certain what the point would be :)